lib/src/http/managers/emoji_manager.dart

```diff
@@ -54,7 +54,7 @@ class EmojiManager extends Manager<Emoji> {
   }
 
   @override
-  FutureOr<GuildEmoji> get(Snowflake id) async => await super.get(id) as GuildEmoji;
+  Future<GuildEmoji> get(Snowflake id) async => await super.get(id) as GuildEmoji;
```

Comment:
why such change?

Reply:
With how they're implemented, all get() methods always return Futures. We should reflect this in the return type so that users can pass them to APIs which need Futures (e.g Future.wait) without needing to wrap them in Future.value or cast them.